test/e2e/networks/apphash-fail.toml

```diff
@@ -0,0 +1,92 @@
+# This file represents an attempt at reproducing a commonly observed failure in 
+# the generated Tendermint e2e test suite. The failure manifests as an node whose
+# AppHash as reported by /status does not agree with the AppHash as reported by the
+# block of the same height in the chain.
+
+ipv6 = true
+initial_height = 1000
+key_type = "secp256k1"
+evidence = 1
+log_level = ""
+queue_type = "priority"
+TxSize = 10240
+
+[initial_state]
+  initial01 = "a"
+  initial02 = "b"
+  initial03 = "c"
+
+[validators]
+
+[validator_update]
+  [validator_update.0]
+    validator01 = 47
+    validator02 = 93
+    validator03 = 79
+  [validator_update.1010]
+    validator04 = 63
+
+[node]
```

Comment:
there are a lot factors here that may be totally random and we should probably work on narrowing down: the db, say. or running abci_protocol as tcp?

there are also a lot of factors that are set to the 0 value here just because of how the generated files are made, that might be better to just delete.

Reply:
Removed the zero values and switched multiple fields to use the same values.